migrations/sql/V2018.11.20.1106__AddModelMeta.sql

```diff
@@ -0,0 +1,60 @@
+--- What this Migration is about
+--- comment on model.is_current
+--- &
+--- creating three new table
+--- 1. model_version_meta
+--- 2. model_version_vaccine
+--- 3. model_version_country
+
+COMMENT ON COLUMN model.is_current IS
+  'TRUE if the model is currently running for VIMC.';
+
+CREATE TABLE model_version_meta (
+  id SERIAL,
+  model_version INTEGER NOT NULL,
+  is_dynamic BOOLEAN NOT NULL, 
+  code TEXT,
+  dalys BOOLEAN NOT NULL,
+  gender INTEGER NOT NULL,
+  burden_min_age INTEGER NOT NULL,
+  burden_max_age INTEGER NOT NULL,
+  cohort_first INTEGER, --- nullable, NA means 2000
+  cohort_last INTEGER, --- nullable, NA means min(2100, age_max+2030)
+  year_min INTEGER, --- nullable, NA means 2000
```

Comment:
we have both `_first`/`_last` and `_min`/`_max` here. Would you like to standardise on one?

Reply:
Done. All with `_min/_max`.